docs/separating-entity-definition.md

```diff
@@ -187,12 +187,58 @@ export const CategoryEntity = new EntitySchema<Category>({
 });
 ```
 
+You can use embedded entities in schema models, like this:
+```ts
+export interface Name {
+    first: string;
+    last: string;
+}
+
+export const NameEntitySchema = new EntitySchema<Name>({
+    name: "name",
+    columns: {
+        first: {
+            type: "varchar"
+        },
+        last: {
+            type: "varchar"
+        }
+    },
+});
+
+export interface User {
+    id: string;
+    name: Name;
+    isActive: boolean;
+}
+
+export const UserEntitySchema = new EntitySchema<User>({
+    name: User.name,
```

Comment:
since `User` is interface `User.name` won't work here. Probably you just need to type "user" here.